bofire/domain/domain.py

```diff
@@ -304,69 +330,9 @@ def get_feature(self, key: str):
         Returns:
             Feature: The feature with the passed key
         """
+        assert isinstance(self.input_features, InputFeatures)
```

Comment:
Same here, why this assert? It is always of type InputFeatures or?

Reply:
As mentioned above, yes, it is. But Pyright cannot analyze the Pydantic conversion if someone passes a list during construction of `Domain`.